frontend/public/components/modals/configure-update-strategy-modal.tsx

```diff
@@ -105,14 +115,20 @@ export const ConfigureUpdateStrategy: React.FC<ConfigureUpdateStrategyProps> = (
                         aria-describedby="input-max-surge-help"
                       />
                       <span className="pf-c-input-group__text">
-                        <Tooltip content="Current desired pod count">
-                          <span>greater than {pluralize(props.replicas, 'pod')}</span>
+                        <Tooltip content={t('modal~Current desired pod count')}>
+                          <span>
+                            {t('modal~greater than {{pods}}', {
+                              pods: pluralize(props.replicas, 'pod'),
+                            })}
```

Comment:
Don't use `pluralize`. Use i18next plurals: https://www.i18next.com/translation-function/plurals

Reply:
use i18n plurals instead of plurals in utils